scripts/lib/build-utils.mjs

```diff
@@ -224,17 +225,33 @@ export default {
    * @param os {"windows" | "linux"}
    */
   async buildNerdctlStub(os) {
-    const platDir = os === 'windows' ? 'win32' : os === 'linux' ? 'linux' : '<unknown>';
-    const basename = os === 'windows' ? 'nerdctl.exe' : 'nerdctl-stub';
-    const outFile = path.join(this.srcDir, 'resources', platDir, 'bin', basename);
+    if (!['windows', 'linux'].includes(os)) {
+      throw new Error(`Unexpected os of ${ os }`);
+    }
+    let platDir, basename, parentDir, outFile, sourceFile, destFile;
 
+    if (os === 'windows') {
+      platDir = 'win32';
+      parentDir = path.join(this.srcDir, 'resources', platDir, 'bin');
+      outFile = path.join(parentDir, 'nerdctl.exe');
+      sourceFile = outFile = path.join(parentDir, 'nerdctl.exe');
```

Comment:
You're assigning to `outFile` twice. It's the same value, so it's fine, but you probably meant to delete one of the two assignments.

Reply:
good catch